.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -32,5 +32,6 @@ assignees: ''
  - Sonarr Branch: <!--[e.g. master, develop , phantom-develop]-->
 
 **Trace Logs**
-Turn on Trace logs under Settings -> General and wait for the bug to occur again. 
-**Upload the full log file here (or another site (e.g. pastebin) and link it). Issues will be closed, if they do not include this!**
\ No newline at end of file
+Turn on Trace logs under Settings -> General and wait for the bug to occur again.
+**Upload the full log file here (or another site (e.g. pastebin) and link it). Issues will be closed, if they do not include this!**
+<!-- If your logs do not say "TRACE" then they are not what is requested.-->
```

Comment:
I think it'd be better to say:

Trace logs are named `Sonarr.trace.txt` or `Sonarr.trace.#.txt`